.gitignore

```diff
@@ -69,3 +69,6 @@ yarn.lock
 
 # Ignore JetBrains IDE file
 .idea
+
+# Tmp build directory
+wcag-act-rules-tmp/
```

Comment:
I accidentally pushed the tmp dir to the repo. Turns out that wasn't in the ignore. It is now.